publication-commons/src/test/resources/features/AccessedToPublishedMaterial.feature

```diff
@@ -0,0 +1,64 @@
+Feature:
+  1. DynamoDBPublicationService allows a user to read any document that is marked as published and
+  therefore it is publicly available.
+
+  2. DynamoDBPublicationService does not allow any user to modify or delete any document that is marked
+  as published.
+
+  Notes:
+  1. By "published" we mean that a document has been published in a Journal, Conference proceedings,
+  or is part of a book and any logged in user can access it.
+
+  2. We assume that when a user is logged in, he/she has read access to all material that is published.
```

Comment:
We assume that when a user is logged in or has role Anonymous, they have read access to all material that is published.